addons/project/models/project.py

```diff
@@ -1665,7 +1665,7 @@ def _compute_access_warning(self):
     @api.depends('child_ids.planned_hours')
     def _compute_subtask_planned_hours(self):
         for task in self:
-            task.subtask_planned_hours = sum(child_task.planned_hours + child_task.subtask_planned_hours for child_task in task.child_ids)
+            task.subtask_planned_hours = sum(child_task.planned_hours for child_task in task.child_ids)
```

Comment:
```suggestion
            task.subtask_planned_hours = sum(task.child_ids.mapped('planned_hours'))
```

Reply:
@auon-odoo 
changes are done. Please take a look whenever you have time.